modules/auras.lua

```diff
@@ -15,7 +15,9 @@ end
 local strfind = string.find
 local pairs, select = pairs, select
 local tinsert, tsort, tremove = table.insert, table.sort, table.remove
-local UnitAura, UnitBuff, UnitDebuff, GetSpellInfo = UnitAura, UnitBuff, UnitDebuff, GetSpellInfo
+local UnitAura = C_UnitAuras and C_UnitAuras.GetAuraDataByIndex or UnitAura

```

Comment:
Those aliases aren't needed anymore, right? Since we use the GladiusEx. variants

Reply:
fixed